### PR TITLE
Add suport for external references

### PR DESCRIPTION
**Pull request by askoretskiy**
Implements feature request mentioned in https://github.com/snok/django-swagger-tester/issues/120

### CODE REVIEW

File: django_swagger_tester/loaders.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def set_schema(self, schema: dict) -> None:
         Sets self.schema and self.original_schema.
         """
         self.schema = self.replace_refs(
-            schema,
+            schema, self
```
Comment (sondrelg) [inline django_swagger_tester/loaders.py:60]:
```suggestion
            schema, loader=self
```

File: django_swagger_tester/loaders.py [hunk 2]
```diff
@@ -232,26 +232,44 @@ def validate_status_code(status_code: Union[int, str]) -> None:
...
-    def replace_refs(schema: dict) -> dict:
+    def replace_refs(schema: dict, loader=None) -> dict:
         """
         Finds all schema references ($ref sections) in an OpenAPI schema and inserts them back into in place of the refs.
         This way we don't have to handle reference section when interacting with a loaded schema.
 
         * This does add a performance overhead to interacting with a schema, so changing this in the future would be fine *
 
         :param schema: OpenAPI schema
+        :param loader: current loader
```
Comment (sondrelg) [inline django_swagger_tester/loaders.py:243]:
```suggestion
        :param loader: Current loader
```

**Review by sondrelg — APPROVED** [review]
Great work @askoretskiy!

I have close to no feedback, and if you've already verified that this solves the problem in your own project as well, I am happy to merge this as is.

Thanks for taking the time to solve this, and feel free to contribute again if you see other features you would like implemented or improved 🏄

**Comment by askoretskiy** [comment]
Thanks for merging it.
Suggestions were good, I would apply them if possible.

Could you please make a patch release @sondrelg ? It would be great if we could install this version via pip:

```
pip install django-swagger-tester==2.1.3
```

**Pull request merged**